macros/cssinfo.ejs

```diff
@@ -11,13 +11,13 @@ var name = $0 || (slug ? slug.split("/").pop().toLowerCase() :
     "preview-wiki-content");
 var atRule = $1;
 
-var propertiesUrl = "https://raw.githubusercontent.com/mdn/data/mdn/css/properties.json";
-var atRulesUrl = "https://raw.githubusercontent.com/mdn/data/mdn/css/at-rules.json";
-var selectorsUrl = "https://raw.githubusercontent.com/mdn/data/mdn/css/selectors.json";
-var typesUrl = "https://raw.githubusercontent.com/mdn/data/mdn/css/types.json";
-var syntaxesUrl = "https://raw.githubusercontent.com/mdn/data/mdn/css/syntaxes.json";
-var unitsUrl = "https://raw.githubusercontent.com/mdn/data/mdn/css/units.json";
-var localStringsUrl = "https://raw.githubusercontent.com/mdn/data/mdn/l10n/css.json";
+var propertiesUrl = "https://raw.githubusercontent.com/mdn/data/master/css/properties.json";
+var atRulesUrl = "https://raw.githubusercontent.com/mdn/data/master/css/at-rules.json";
+var selectorsUrl = "https://raw.githubusercontent.com/mdn/data/master/css/selectors.json";
+var typesUrl = "https://raw.githubusercontent.com/mdn/data/master/css/types.json";
+var syntaxesUrl = "https://raw.githubusercontent.com/mdn/data/master/css/syntaxes.json";
+var unitsUrl = "https://raw.githubusercontent.com/mdn/data/master/css/units.json";
+var localStringsUrl = "https://raw.githubusercontent.com/mdn/data/master/l10n/css.json";
```

Comment:
Tested on [Web/CSS/outline-offset](https://developer.mozilla.org/en-US/docs/Web/CSS/outline-offset). Renders the same, except that the initial value is no longer wrapped in a ``<code>`` element.